client/signup/steps/user/index.jsx

```diff
@@ -62,7 +62,7 @@ export class UserStep extends Component {
 			// Hides sub header for this particular flow
 			subHeaderText = '';
 		} else if ( flowName === 'wpcc' && oauth2Client ) {
-			if ( oauth2Client.id === 50916 ) {
+			if ( oauth2Client.name === 'woo' ) {
```

Comment:
I think it's safer to stick with IDs, rather than compare it by name. Name currently isn't returned by the client-data endpoint, but I don't want to rely on that.

I'd prefer the failure state to be that we could accidentally show WordPress.com branding, rather than accidentally show Woo branding.